src/views/InProgress.vue

```diff
@@ -528,8 +528,15 @@ export default defineComponent({
 
       // TODO: update alert message when itemsToReject contains a single item and also in some specific conditions
       let message;
-      if(!outOfStockItem) {
-        message = this.$t('Are you sure you want to perform this action?')
+      if (!outOfStockItem) {
+        const firstItem = itemsToReject[0];
+        if (itemsToReject.length === 1) {
+          if (firstItem.rejectReason === "REJ_RSN_DAMAGED") message = this.$t('is identified as damaged. This order item will be unassigned from the store and sent to be rebrokered.', { productName: firstItem.productName });
+          else if (firstItem.rejectReason === "WORN_DISPLAY") message = this.$t('is identified as worn display. This order item will be unassigned from the store and sent to be rebrokered.', { productName: firstItem.productName });
+          else if (firstItem.rejectReason === "MISMATCH") message = this.$t('is identified as mismatched. This order item will be unassigned from the store and sent to be rebrokered.', { productName: firstItem.productName });
```

Comment:
Instead of three label entries, we can pass the reject reason string as a parameter and hence, only a single label will be required.

Reply:
done